Allow admins to see user profiles

devchix opened the pull request:
https://trello.com/c/H18dkHE2

Adds ability for admin to see user email in index view as well as profile information on the donor show page which previously only showed the donor donation history.

Screens shots:
<img width="916" alt="screen shot 2016-07-07 at 3 18 10 pm" src="https://cloud.githubusercontent.com/assets/13186/16670511/cce34022-4457-11e6-8067-c658d96dd265.png">

<img width="685" alt="screen shot 2016-07-07 at 3 18 20 pm" src="https://cloud.githubusercontent.com/assets/13186/16670515/d296072a-4457-11e6-8832-7b8c1671f379.png">

seanpdoyle commented on app/helpers/donors_helper.rb:
```diff
@@ -0,0 +1,5 @@
+module DonorsHelper
+  def is_or_isnt_on_site(donor)
+    donor.location.grown_on_site ? "is" : "is not"
```
From [thoughtbot/guides](https://github.com/thoughtbot/guides/blob/83cfbe7327d574efc1a8943c9e1f60b9af751e1b/style/ruby/README.md):

> Avoid ternary operators (boolean ? true : false). Use multi-line if instead to emphasize code branches.

seanpdoyle replied:
`"is"` and `"is not"` isn't going to work in other languages (in the future), and sort of defeats the purpose of internationalization.

What do you think about having two separate I18n values -- one for the true case and one for the false case?

seanpdoyle commented on app/views/donors/show.html.erb:
```diff
@@ -1,7 +1,26 @@
+      <%= t(".grow_information") %>
+    </h2>
+    <span><%= t(".grow_information_sentence",
+            is_is_not: is_or_isnt_on_site(@donor),
```
What do you think about another I18n key that is more descriptive and less redundant?

What do you think about `on_site`?

devchix commented:
@seanpdoyle Thanks for the feedback. Internationalization is something I don't have a ton of experience with so it is still kind of awkward to me. Thanks for the feedback. I think I addressed the comments. Can you take another look?

KioKrofovitch commented on app/helpers/donors_helper.rb:
```diff
@@ -0,0 +1,9 @@
+      t(".grown_on_site")
+    else
+      t(".grown_of_site")
+    end
```
Should that be `grown_off_site`?

seanpdoyle commented on app/helpers/donors_helper.rb:
```diff
@@ -0,0 +1,9 @@
+module DonorsHelper
+  def on_site?(donor)
+    if donor.location.grown_on_site
```
Is `Location#grown_on_site` a boolean? 

If so, what do you think about using the `#grown_on_site?` version to communicate that it's a predicate?
